src/pudl/metadata/resources/eia861.py

```diff
@@ -476,6 +600,13 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_operational_data_revenue": {
```

Comment:
This is another table I think we might rename along the lines of FERC 1 to make their similarity clear. Given that we have `out_ferc1__yearly_operating_revenues_sched300` maybe

- `out_eia861__yearly_operating_revenues`

Reply:
See comment from above